src/Markdig.Wpf/Renderers/Xaml/Inlines/EmphasisInlineRenderer.cs

```diff
@@ -36,7 +36,25 @@ public EmphasisInlineRenderer()
         protected override void Write([NotNull] XamlRenderer renderer, [NotNull] EmphasisInline obj)
         {
             var tag = GetTag(obj);
-            renderer.Write("<").Write(tag).Write(">");
+            renderer.Write("<").Write(tag);
+            switch (obj.DelimiterChar)
+            {
+                case '~':
```

Comment:
Just for the sake of completeness, add the cases for `'*'` and `'~'` that immediately break. It keeps balance with the previous method and shows that those two options are valid.